systems/plants/RigidBody.cpp

```diff
@@ -146,6 +148,60 @@ bool RigidBody::hasParent() const {
   return parent !=nullptr;
 }
 
+void RigidBody::setCollisionFilter(const DrakeCollision::bitmask& group, 
+                                   const DrakeCollision::bitmask& mask)
+{
+  setGroup(group);
+  setMask(mask);
+}
+
+bool RigidBody::appendCollisionElementIds(const string& group_name, vector<DrakeCollision::ElementId>& ids) const
+{
+  auto group_ids_iter = collision_element_groups.find(group_name);
+  if (group_ids_iter != collision_element_groups.end()) {
+    ids.reserve(ids.size() + distance(group_ids_iter->second.begin(), group_ids_iter->second.end()));
+    ids.insert(ids.end(), group_ids_iter->second.begin(), group_ids_iter->second.end());
+    return true;
+  } else {
+    return false;
+  }
+}
+
+bool RigidBody::appendCollisionElementIds(vector<DrakeCollision::ElementId>& ids) const
```

Comment:
Can we document/rename this one?  On quick inspection, I expected it to take ids and append them to the body collision_element_ids, but it does the opposite.